### PR TITLE
Added page on how to add to the blog

### PR DESCRIPTION
**Pull request by niallrobinson**
A two speed article on how to contribute to the blog. Jacob - I hadn't twigged that you'd written the wiki article on blogging yourself until I finished this, but I thought it might be good to dumb it down for scientists anyway :wink:

### CODE REVIEW

**Comment by niallrobinson** [comment]
@alberto-arribas After talking to @jacobtomlinson, we've decided to streamline the workflow here a little bit. Having the `_drafts` folder seems unnecessarily complicated. I'll post back here when I've updated it

**Comment by niallrobinson** [comment]
Alright - go nuts!

**Comment by jacobtomlinson** [comment]
Made a couple of tweaks. Let me know if you're happy with them and then I'll merge.

**Comment by niallrobinson** [comment]
I'm pretty happy with this now. We should probably let @alberto-arribas take a peek before merging?

**Comment by jacobtomlinson** [comment]
@alberto-arribas if you want to test it out find your local copy in Terminal and run the following

```
git fetch origin
git checkout -b how_to_blog origin/how_to_blog
jekyll serve
```

and then visit http://127.0.0.1:4000/getting-started/how-to-blog

**Comment by niallrobinson** [comment]
Also, you can view in github by looking at the Files Changed tab and clicking view. The rendering is less perfect but it should do for quickly reading content.

**Comment by jacobtomlinson** [comment]
:+1:

**Comment by niallrobinson** [comment]
Merging for the time being
